src/main/java/data/SheetsConverter.java

```diff
@@ -30,29 +30,67 @@
 public class SheetsConverter {
   private static Sheets sheetsService;
   private static final String SPREADSHEET_ID = "1QnVlh-pZHycxzgQuk0MN2nWOY6AGu9j4wGZaGzi_W9A";
-  private final String ACCOUNT_TITLE = "Account Sheet";
-  private final String VENDOR_TITLE = "Vendor Sheet";
+  private final String ACCOUNT_RANGE = "'sheet2'!A1";
+  private final String VENDOR_RANGE = "'sheet1'!A1";
+  private final String INPUT_OPTION = "RAW";
 
   public boolean updateSheets(ArrayList<Vendor> vendorList, String accessToken) 
     throws GeneralSecurityException, IOException {
-    writeToSheet(accessToken); 
+    writeToSheet(vendorList, accessToken); 
     return true;
   }
 
-  // Enter unimportant data to a spreadsheet.
-  public static void writeToSheet(String accessToken) throws GeneralSecurityException, IOException {
+  public void writeToSheet(ArrayList<Vendor> vendorList, String accessToken) throws GeneralSecurityException, IOException {
     sheetsService = SheetsServiceUtil.getSheetsService(accessToken);
+    ArrayList<ArrayList<Account>> allAccounts = getAllAccounts(vendorList); 
 
-    ValueRange body = new ValueRange().setValues(Arrays.asList(
-      Arrays.asList("Expenses January"), 
-      Arrays.asList("books", "30"), 
-      Arrays.asList("pens", "10"),
-      Arrays.asList("Expenses February"), 
-      Arrays.asList("clothes", "20"),
-      Arrays.asList("shoes", "5")));
-    UpdateValuesResponse result = sheetsService.spreadsheets().values()
-      .update(SPREADSHEET_ID, "A1", body)
-      .setValueInputOption("RAW")
-      .execute();
+    // Retrieve the latest spreadsheet data.
+    ValueRange accountBody = new ValueRange().setValues(
+      buildAccountSheetBody(allAccounts));
+    ValueRange vendorBody = new ValueRange().setValues(
+      buildVendorSheetBody(vendorList));
+
+    // Update the Account and Vendor spreadsheets using the update data.
+    UpdateValuesResponse updateAccountSheet = sheetsService.spreadsheets()
+      .values().update(SPREADSHEET_ID, ACCOUNT_RANGE, accountBody)
+      .setValueInputOption(INPUT_OPTION).execute();
+    UpdateValuesResponse updateVendorSheet = sheetsService.spreadsheets()
+      .values().update(SPREADSHEET_ID, VENDOR_RANGE, vendorBody)
+      .setValueInputOption(INPUT_OPTION).execute();
+  }
+
+  /** Gets every Account that is associated with any Vendor from the List. */
+  public ArrayList<ArrayList<Account>> getAllAccounts(ArrayList<Vendor> vendors) {
+    ArrayList<ArrayList<Account>> allAccounts =  new ArrayList<ArrayList<Account>>();
+
+    for(int i = 0; i < vendors.size(); i++) {
+      allAccounts.add(vendors.get(i).getAccounts());
+    }
+
+    return allAccounts; 
+  }
+
+  public List buildAccountSheetBody(ArrayList<ArrayList<Account>> accounts) {
```

Comment:
`List<List<String>>`?

Reply:
Please be consistent about the typing for your lists. Probably best to use ArrayList everywhere

Reply:
When I try to make everything and ArrayList, I run into conversion errors saying I can't change an ArrayList<ArrayList> to a List<List>.

Reply:
At the very least, we need to specify the template parameters for the List
(i.e. `List<String>` instead of `List`).

That said, we should be able to implicitly convert an ArrayList to a List, since ArrayList is a subclass of List. Going the other way is where you may run into trouble.